src/ui/pages/SendTransaction/SendTransaction.tsx

```diff
@@ -83,6 +94,186 @@ import { TransactionAdvancedView } from './TransactionAdvancedView';
 import { TransactionWarnings } from './TransactionWarnings';
 import { txErrorToMessage } from './shared/transactionErrorToMessage';
 
+function assertTransactionProps(
```

Comment:
May be we can move these function to separate files here? Just to make diff for this file less scary :)

Reply:
https://github.com/zeriontech/zerion-wallet-extension/pull/468#discussion_r1620464453